index.html

```diff
@@ -226,5 +226,15 @@ <h3>Skills</h3>
                                 </li>
                             </ul>                 
       </section>
+      <section class="contact-form">
+        <h2 class="contact-me">Contact me</h2>
+        <p class="contact-paragraph">If you have an application you are interested in developing, a feature that you need to built or a project that needs coding. I'd love to help with it!</p>
+        <form action="https://formspree.io/ivonnebenites2015@gmail.com" method="post" class="form">
+            <input type="text" name="name" id="name" placeholder="name" class="input">
+            <input type="email" name="email" id="email" placeholder="email" class="input">
+            <textarea name="description" id="" cols="30" rows="10" placeholder="Write your message here.." class="input-text"></textarea>
+            <input type="submit" value="Get in touch" class="button-contact-form">
+        </form>
```

Comment:
Great job with your form! However, there are a few friendly suggestions to make it even better and more functional:

- [ ] Could you please double-check the link? It seems that it's not set up correctly. **You'll need to sign up in formspree and grab the correct ID from there.** It should be similar to this format: `https://formspree.io/f/{form_id}`.

- [ ] Also, please ensure that the name input allows a maximum of 30 characters, as stated in the project requirements.

- [ ] Additionally, it would be great if you could **set a character limit of 500 for the textarea**, as specified in the project requirements.

- [ ] Finally, it would be helpful if **you could add the "required" attribute to all the inputs**. This way, users won't be able to submit the form if any field is left empty. Thanks for considering these suggestions!

   ### Error trying to send your form:
   ![image](https://github.com/IvonneBenitesRodriguez/microverse_mobile_portfolio/assets/107952724/3e746896-50e7-402c-b373-67013ce512e5)

Reply:
Dear @taldr27 👋🏾
Thank you for your suggestions and also for giving a clear explanation about the usage of required attribute in the inputs and also about the format of how to use the formspree 🙏🏿
The changes you suggested are done, please verify it, 
Best regards,

Reply:
Hello👋🏾 @IvonneBenitesRodriguez, your changes are looking good, don't forget to add required to all your inputs, including the `text-area` . Happy coding.

Reply:
Dear @taldr27 
Thank you to check the project  🙏🏿,
now the required to text-area is added,
Again thank you for this suggestion, 
have a good day @taldr27